app/app-core.sh

```diff
@@ -105,7 +105,7 @@ app_install_core()
 
     rm -rf "$CONFIG_PATH_MAINNET" "$CONFIG_PATH_DEVNET" "$CONFIG_PATH_TESTNET" "$BRIDGECHAIN_PATH"
 
-    git clone https://github.com/ArkEcosystem/core.git --branch 2.3.22 "$BRIDGECHAIN_PATH"
+    git clone https://github.com/ArkEcosystem/core.git --branch 2.5.14 "$BRIDGECHAIN_PATH"
```

Comment:
```suggestion
    git clone https://github.com/ArkEcosystem/core.git --branch 2.5.19 "$BRIDGECHAIN_PATH"
```

Or not? :thinking:

Reply:
yes, thanks! this hasn't been updated since .14 👌